dist/browser/bootlint.js

```diff
@@ -11343,6 +11394,34 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
+        var FUTURE_VERSION_ERROR = "Detected what appears to be Bootstrap v4 or later. This version of Bootlint only supports Bootstrap v3.";
```

Comment:
Out of curiosity, was the choice to go with 'FUTURE…' because Bootlint will only support V3 long term or is planning for supporting both now just pre-mature optimization? Thanks :smiley: